journeyman/javascript/flow.md

```diff
@@ -0,0 +1,34 @@
+# JavaScript Basics
+
+## Intro
+
+With linting we can easily find syntax errors and poor code. However, there is another type of error which plagues JavaScript due to it's dynamically typed variables: Type Errors. Type errors happen when you attempt to use one type as if it were another type. For example, trying to access properties on `undefined`. It doesn't work.
```

Comment:
"For example, trying to access properties on `undefined`. It doesn't work."  Please re-word